devel/robodoc/Portfile

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                robodoc
-version             4.99.41
-revision            1
+github.setup        gumpu robodoc 4.99.44 v
```

Comment:
There are different ways one can get the source from a GitHub project and these are `releases`, `archive`, and `tarball` (an explanation can be found [here](https://github.com/macports/macports-ports/pull/2587#issuecomment-423378492))

In short it boils down to: (1) if the author manually creates/uploads a release we should use that (i.e., `releases`). Otherwise I think think `archive` is preferred over `tarball`, which is currently still set as the default but mainly to avoid breaking all the current ports as the `checksums` will change. So when updating a port that uses GitHub to a new version it would be good to try to fetch either from `releases` or `archive`.

I don't think upstream uses `releases` here, so the next-best-thing would be `archive` - so could you add the line `github.tarball_from archive`? You would need to update the `checksum` though as they will change (i.e., `sudo port clean --all robodoc && sudo port -v checksum robodoc`).

Reply:
Thanks @reneeotten for your advice. I was not aware of these distinctions. I.e., it worked, so I did not think further.
Indeed the github project does not seem to do releases, so `archive` is the best option. I'm not sure this is going to chance the checksum (as the same file would be retrieved), but I'll check.

Then I need to figure out how to squash the two commits. I never found any authoritative answer to this.